alarms/Telegram/telegram_alarm.py

```diff
@@ -84,7 +108,7 @@ def send_alert(self, alert, info):
 				'disable_notification': 'True'
 			}
 			try_sending(log, self.connect, "Telegram (Loc)", self.client.sendLocation, locargs)
-			
+
 	#Trigger an alert based on Pokemon info
 	def pokemon_alert(self, pokemon_info):
 		self.send_alert(self.pokemon, pokemon_info)
```

Comment:
Change the alerts to look something like this:
```
if self.pokemon.stickers:
    self.send_alert(self.pokemon, pokemon_info, stickerlist[info['id'])
elif:
    self.send_alert(self.pokemon, pokemon_info)
```

If stickers are enabled, add the sticker to the alert call, otherwise leave it off. For gyms and pokestops do stickerlist['gym'] or stickerlist['pokestop']